File: computingservices/DocumentServices/utils/dbconnection.py
import psycopg2
from . import docservice_db_user,docservice_db_port,docservice_db_host,docservice_db_name,docservice_db_password

def getdbconnection():
    conn = psycopg2.connect(
        host=docservice_db_host,
        database=docservice_db_name,
        user=docservice_db_user,
        password=docservice_db_password,port=docservice_db_port)
    return conn    

File: computingservices/CompressionServices/utils/dbconnection.go
package utils

import (
	"database/sql"
	"fmt"
	"log"

	_ "github.com/lib/pq"
)

func GetDBConnection() *sql.DB {
	//cfg := LoadConfig()

	host := ViperEnvVariable("COMPRESSION_DB_HOST")
	port := ViperEnvVariable("COMPRESSION_DB_PORT")
	user := ViperEnvVariable("COMPRESSION_DB_USER")
	password := ViperEnvVariable("COMPRESSION_DB_PASSWORD")
	dbname := ViperEnvVariable("COMPRESSION_DB_NAME")

	psqlInfo := fmt.Sprintf(
		"host=%s port=%s user=%s password=%s dbname=%s sslmode=disable",
		host, port, user, password, dbname,
	)
	//fmt.Printf("psqlInfo:%v\n", psqlInfo)
	db, err := sql.Open("postgres", psqlInfo)
	if err != nil {
		log.Fatalf("Error connecting to DB: %v", err)
	}
	return db
}
